app/assets/javascripts/student/interventions_controller_templates.hbs

```diff
@@ -0,0 +1,125 @@
+{{#main}}
+  <div class="left-panel">
+    <div class="add-new-intervention {{addNewClassName}}">
+      <h1>
+        <span class="add-intervention-plus-sign">
+          +
+        </span>
+        <span class="add-intervention-text">
+          Add intervention
+        </span>
+      </h1>
+    </div>
+    <div class="intervention-cell-list"></div>
+  </div>
+  <div class="right-panel">
+    <div class="intervention-details-list"></div>
+    <div class="new-intervention-container"></div>
+  </div>
+{{/main}}
+    
+{{#interventionCell}}
```

Comment:
This could be broken out into multiple files, but it seemed more cohesive to keep them in the same file like before.  So at render time, callers pass in `{ interventionCell: .. }` or whichever key to indicate what piece they want.